lms/static/js/commerce/views/receipt_view.js

```diff
@@ -19,11 +19,6 @@ var edx = edx || {};
             this.useEcommerceApi = this.ecommerceBasketId || this.ecommerceOrderNumber;
             _.bindAll(this, 'renderReceipt', 'renderError', 'getProviderData', 'renderProvider', 'getCourseData');
 
-            /* Mix non-conflicting functions from underscore.string (all but include, contains, and reverse) into
-             * the Underscore namespace.
-             */
-            _.mixin(_s.exports());
```

Comment:
You could also add:

    window._.str = str;

I suppose. 

Note that we are working to get rid of Underscore.string because it is mostly not needed with the latest browsers. My recommendation would be to remove this line, and to try to remove any usages of it in your code. @bjacobel might be able to offer some advice as he's worked on this recently.